### PR TITLE
DRA-853 - new facets on search result page

### PR DESCRIPTION
**Pull request by jesperlauridsen**
**--- DEPENDING ON DRA-855 ---**

The next iteration of the facet rework for the seachpage.

- Added the datepicker for time searching on the facets.
- Did a lot of small refactoring to make everything work together nicely
- Updated the sort mechanism for now
- Styled the top Overhead bar a little more according to the design
- Added a few more translations
- Made sure startTime search is working across the TimeSearchFilters.vue searches.

### CODE REVIEW

**Review by jorntx — COMMENTED** [review]
Overall this looks good I have only a few code comments.

Discussion points:
* On the search page when you unfold the filters you have to unfold again to see the complete list of channels. I get that we have to restrict the amound of channels shown but I think we could show some more channels pr. default.
* The 'Udvælg kanaler' graphic/text, 'Ryd filtre' graphic/text and 'TV' & 'Radio' & 'Dato og tid' toggle buttons are large and appear very verbose on my 13 inch screen.
* A usability test should decide in what order the date/time and channel filters should appear. Now it is date/time first (although disabled pr. default).

File: src/components/search/SearchOverhead.vue [hunk 1]
```diff
@@ -98,6 +149,96 @@ export default defineComponent({
...
 		const route = useRoute();
 		const { t } = useI18n();
+		const toggleFacetsButton = ref<HTMLButtonElement | null>(null);
+		const tvToggled = ref(true);
+		const radioToggled = ref(true);
+
+		const delimitationOptions = {
+			all: '',
+			tv: 'origin:"ds.tv"',
+			radio: 'origin:"ds.radio"',
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/search/SearchOverhead.vue:159]: I've decided to leave this for now. Imho we can elaborate on it later, if it's needed.

File: src/components/common/TimeSearch/TimeSearchFilters.vue [hunk 2]
```diff
@@ -227,63 +265,84 @@ export default defineComponent({
...
-			APIService.getFullResultWithFacets().then((reponse) => {
-				const resultsInYears = reponse.data.facet_counts.facet_fields.temporal_start_year;
-				const sortedResults = [] as dataItem[];
-				resultsInYears.forEach((item, index) => {
-					if (index % 2 === 0) {
-						let nextResult = {
-							year: item,
-							items: Number(resultsInYears[index + 1]),
+			if (props.timeline) {
+				APIService.getFullResultWithFacets().then((reponse) => {
```
[the remark this answers is not in the document]
Comment (jesperlauridsen) [reply src/components/common/TimeSearch/TimeSearchFilters.vue]: I've done some stuff now. It's the best we can do for now - but if the backend fails here, we got bigger problems than the slider! ;-)

**Review by jorntx — APPROVED** [review]
Ready to merge!

**Pull request merged**